src/tlo/methods/epi.py

```diff
@@ -62,17 +62,17 @@ class Epi(Module):
                                     "whether all doses have been received of the tetanus/diphtheria vaccine"),
     }
 
-    def __init__(self, name=None, resourcefilepath=None):
+    def __init__(self, name=None):
         # NB. Parameters passed to the module can be inserted in the __init__ definition.
         super().__init__(name)
-        self.resourcefilepath = resourcefilepath
+
         self.all_doses = dict()
         self.cons_item_codes = dict()  # (will store dict giving item_codes for each vaccine)
 
-    def read_parameters(self, data_folder):
+    def read_parameters(self, resourcefilepath=None):
```

Comment:
Not quite. We are setting it to `None` to allow flexibility since we have cases where the parameter resourcefilepath is not needed but we still need to call the function. One example is in `chronicsyndrome.py` line 98 (function name is also `read_parameters`).

Reply:
Okay. I see. 
Are there other examples?

Chronic_syndrome is a funny one because it's just a module used for "teaching" and "testing" and probably does lots of weird stuff!! If so, we should change chronic syndrome rather than adapt our framework design to suit it. (Different to other modules where I know it's right that we go out of our way not to disturb them!)

Anyway, it's a very small point about whether we should have a default specified or not, and asked mostly for my own interest. 

Thanks